server/src/callbacks.mjs

```diff
@@ -75,11 +76,20 @@ Empirica.onNewBatch(async function ({ batch }) {
     try {
       console.log("fetching");
       const fetched = await (await axios.get(url)).data;
+      try {
+        marked.parse(fetched);
+      } catch (error) {
+        console.log("Unable to parse markdown");
+      }
+      if ((fetched.match(new RegExp("\\S", "g")) || []).length < 75) {
+        console.warn("Detected under 75 characters in the topic markdown - please check that your file was loaded properly")
+        console.log("Detected under 75 characters in the topic markdown - please check that your file was loaded properly");
```

Comment:
Instead of logging the same error message here, can you just log `fetched` so that under the warning it'll print what it found?